src/frequenz/client/dispatch/_client.py

```diff
@@ -123,9 +123,12 @@ async def create(
         dry_run: bool = False,
         payload: dict[str, Any] | None = None,
         recurrence: RecurrenceRule | None = None,
-    ) -> None:
+    ) -> Dispatch:
         """Create a dispatch.
 
+        Will try to return the created dispatch, identifying it by
+        the same fields as the request.
```

Comment:
Maybe present tense is nicer, but you could also elaborate a bit on what "try" means:

```suggestion
        Returns the created dispatch, if it can be identified using
        the same fields as the request.  If not, raises an exception.
```

Reply:
That's a fair suggestion, though given that we'll update this part soon, I think we can leave it for now.